lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -107,6 +107,10 @@ function dosomething_helpers_preprocess_partners_vars(&$vars) {
       if (!empty($image_node->field_image_landscape)) {
         $img = dosomething_image_get_themed_image($image_node->nid, 'landscape', '550x300');
         $vars['partner_info'][$delta]['image'] = $img;
+      } else {
```

Comment:
No, without that the image stays an object, see https://github.com/DoSomething/phoenix/blob/2b6cf2c850c5e2b0fa91de0f92e97f92d33f4307/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc#L108-L109.

This caused fatal error in #4916.